src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+interface AreaCalculator {
+    double getArea();
+}
```

Comment:
According to the checklist, class and interface names should not be verbs. Consider renaming `AreaCalculator` to something that represents its purpose without using verb forms, such as `AreaComputable` or `AreaProvider`.